### DISCUSSION TITLE
goldendict-ng vs. goldendict?

### DISCUSSION THREAD

**Discussion by doctorcolossus** [xiaoyifang/goldendict-ng#1527] [General]
Hi, I just noticed this project and am just wondering about the motivation for this fork, since [goldendict](https://github.com/goldendict/goldendict) is still alive, isn't it? Why the decision to fork, rather than contribute? Just curious.

Also, one small piece of feedback. I was curious about the Anki integration and was about to ask whether there was any documentation, but I accidentally found it by clicking on one of the screenshots in the `README.md`. I would suggest adding a specific link entitled "documentation", which is clear and easy for people to find. Right now it is not obvious how to get there.

**Comment by shenlebantongying** [comment]
>  Anki integration and was about to ask whether there was any documentation,

Yes, there is doc https://xiaoyifang.github.io/goldendict-ng/topic_anki/

**Comment by doctorcolossus** [comment]
There was only one question, which is the one which you wrote that you don't want to "waste time" answering. Okay then.

As I wrote, I was able to find the documentation, but with difficulty, so I was not asking whether documentation exists, but offering a suggestion to make it easier to find. If you visit the `README.md` and search for the word "documentation", there are no results. I stumbled upon the documentation by clicking on one of the screenshots, which is confusing. My suggestion would be to make an explicit reference to the documentation from the `README.md`,  so that others can find it more easily.
